### PR TITLE
Removing bcgov, adding new ux config

### PR DESCRIPTION
**Pull request by usingtechnology**
added a new property to override the genesis url
updated aca-py version
DID NOT update release or image tags - unsure what is going on with new builds.

This works with commit [fc68763](https://github.com/hyperledger-labs/business-partner-agent/commit/fc687636b956046fa63e5751a9c8b9601e643d2b)

Signed-off-by: Jason Sherman <jsherman@parcsystems.ca>

### CODE REVIEW

**Comment by etschelp** [comment]
Sorry, I just saw that there is a PR waitiung. See comments.

File: charts/bpa/README.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Note: Deleting the PVC's will delete postgresql data as well. Please be cautious
 | acapy.fullnameOverride | string | `""` |  |
 | acapy.image.pullPolicy | string | `"IfNotPresent"` |  |
 | acapy.image.repository | string | `"bcgovimages/aries-cloudagent"` |  |
-| acapy.image.tag | string | `"py36-1.16-1_0.7.0"` | Overrides the image tag whose default is the chart appVersion. |
+| acapy.image.tag | string | `"py36-1.16-1_0.7.1"` | Overrides the image tag whose default is the chart appVersion. |
```
Comment (etschelp) [inline charts/bpa/README.md:185]: already on 0.7.2-rc0 on main, so can be reverted

File: charts/bpa/templates/_helpers.tpl [hunk 2]
```diff
@@ -93,10 +93,18 @@ generate hosts if not overriden
 generate ledger browser url
 */}}
 {{- define "bpa.ledgerBrowser" -}}
-{{- $ledgerBrowser := dict "bosch-test" "https://indy-test.bosch-digital.de" "idu" "https://explorer.idu.network" "bcovrin-test" "http://test.bcovrin.vonx.io" -}}
```
Comment (etschelp) [inline charts/bpa/templates/_helpers.tpl:96]: Is that one outdated or should'nt be used, or is there a better option? I added that one to have another test option.

File: charts/bpa/values.yaml [hunk 3]
```diff
@@ -194,39 +195,32 @@ ux:
...
-#        alt: ""
-#        images: []
 
 acapy:
   image:
     repository: bcgovimages/aries-cloudagent
     pullPolicy: IfNotPresent
     # --  Overrides the image tag whose default is the chart appVersion.
-    tag: py36-1.16-1_0.7.0
+    tag: py36-1.16-1_0.7.1
```
Comment (etschelp) [inline charts/bpa/values.yaml:223]: already on 0.7.2-rc0 on main, so can be reverted

**Review by etschelp — CHANGES_REQUESTED** [review]

File: charts/bpa/templates/_helpers.tpl (hunk 2 above)
Comment (parc-jason) [reply charts/bpa/templates/_helpers.tpl:96]: not outdated, was just cleaning up any bcgov specific code from this repository, it just needs to reside in our fork.

Comment (etschelp) [reply charts/bpa/templates/_helpers.tpl:96]: I think its ok to keep bcovrin-test as an option, basically any ledger that allows public registration can be added here.

**Comment by etschelp** [comment]
merged with https://github.com/hyperledger-labs/business-partner-agent-chart/pull/44

**Pull request closed without merging**